internal/server/instance/drivers/driver_qemu.go

```diff
@@ -3384,25 +3384,28 @@ func (d *qemu) generateQemuConfigFile(cpuInfo *cpuTopology, mountInfo *storagePo
 
 	cfg = append(cfg, qemuTablet(&tabletOpts)...)
 
-	// Existing vsock ID from volatile.
-	vsockID, err := d.getVsockID()
-	if err != nil {
-		return "", nil, err
-	}
+	// Windows doesn't support virtio-vsock
+	if !strings.Contains(d.expandedConfig["image.os"], "Windows") {
```

Comment:
I'd get the value through `strings.ToLower()` to avoid `Windows` then not being considered as windows.